<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>InstallAppWindow</class>
    <widget class="QMainWindow" name="InstallAppWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>741</width>
                <height>543</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Miledger - Installation</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                    <widget class="QGroupBox" name="groupBox">
                        <property name="autoFillBackground">
                            <bool>true</bool>
                        </property>
                        <property name="title">
                            <string>Install Progress</string>
                        </property>
                        <layout class="QGridLayout" name="gridLayout_2">
                            <item row="0" column="0">
                                <layout class="QFormLayout" name="formLayout">
                                    <item row="0" column="0">
                                        <widget class="QPushButton" name="btnInstall">
                                            <property name="text">
                                                <string>Install</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="0" colspan="2">
                                        <widget class="QTextEdit" name="installLog">
                                            <property name="autoFillBackground">
                                                <bool>false</bool>
                                            </property>
                                            <property name="undoRedoEnabled">
                                                <bool>false</bool>
                                            </property>
                                            <property name="readOnly">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="1">
                                        <widget class="QProgressBar" name="progressInstall">
                                            <property name="maximum">
                                                <number>1</number>
                                            </property>
                                            <property name="value">
                                                <number>0</number>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>741</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
